etc/config-template.toml

```diff
@@ -56,7 +56,7 @@
 
 [readpool.storage]
 ## Whether to use the unified read pool to handle storage requests.
-# use-unified-pool = false
+# use-unified-pool = true
```

Comment:
Now same as 
https://github.com/tikv/tikv/blob/master/etc/config-template.toml#L86

Reply:
Yes, it's now on by default.